docs/version0.15.md

```diff
@@ -70,6 +71,16 @@ For compatibility, the [`-XX:OnOutOfMemoryError`](xxonoutofmemoryerror.md) OpenJ
 
 The VM now supports the allocation of huge pages on Linux when you use the `madvise` (`/sys/kernel/mm/transparent_hugepage/enabled`) setting. To enable this feature, set [`-XX:+TransparentHugePage`](xxtransparenthugepage.md) on the command line when you start your application. This option is currently not enabled by default.
 
+###  ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) Support for low-overhead heap profiling
+
+[JEP 331](http://openjdk.java.net/jeps/331) provides a mechanism for sampling Java heap allocations with a low overhead via
+the JVM Tool Interface (JVMTI).
+
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restrictions:** JEP 331 is implemented for OpenJ9 with the following limitations:
+
+- The `balanced` and `metronome` garbage collection policies are not supported.
+- The changes might not be compatible with the Health Center agent, which uses the same sampling interval.
```

Comment:
"might not" seems strange. Can't we figure out whether they are incompatible or not?
@JasonFengJ9

Reply:
Depends. Health Center agent uses same sampling interval. If `JEP331 JVMTI` agent happens to have same interval values as the Health Center agent, there is no problem. Also if the Health Center agent just displays the sampling allocation without analysis associated with the interval value, there is no problem either.

However if the Health Center agent does rely on the sampling interval value initially passed via command line option, and `JEP331 JVMTI` agent changes it afterwards, the result will be wrong.

P.S. @charliegracie mentioned the sampling before `JEP331` wasn't accurate, a fix is expected via https://github.com/eclipse/openj9/issues/6024.

Reply:
I'm thinking we should clarify the release note with this information. "might not" makes it seem like we didn't know.

Reply:
Agreed. How about following:
`JEP331 JVMTI` agent uses same sampling interval as the Health Center agent. If both agents expect same interval value, there is no incompatibility, otherwise the Health Center agent will get incorrect results.

Reply:
Yes, much better.

Reply:
How about - JEP331 JVMTI agent *by default* uses same sampling interval as the Health Center agent.

Reply:
> JEP331 JVMTI agent by default uses same sampling interval as the Health Center agent.

Health Center agent passes the interval value via a command line option, if no such value is passed, `16M` is the default sampling interval. 
`JEP331 JVMTI` agent changes the sampling interval to `512K` by default when it is enabled.
So the default sampling interval is different.

Reply:
Changes made.

Reply:
So you are saying that if the JEP 331 agent and the health center agent are used at the same time, then Health Center won't work? 

The JEP331 JVMTI agent and the Health Center agent both set a sampling interval, which by default is different. If both agents are used at the same time, the Health Center agent will get incorrect results, unless the sampling intervals are adjusted to use the same value.

Reply:
Text updated.

Reply:
Waiting for Jason to approve before merging.

Reply:
Yeah, it looks good.